src/microbe_stage/editor/EndosymbiosisPopup.cs

```diff
@@ -8,6 +8,8 @@
 /// </summary>
 public partial class EndosymbiosisPopup : CustomWindow
 {
+    public bool Lawk;
```

Comment:
I think this should be marked as [Export] because otherwise this is just a random public field (which is not optimal for OOP style).